tests/conftest.py

```diff
@@ -106,7 +107,8 @@ def valid_simple_ml_predictor_data():
             latent_variables=[y.dump()],
             training_data=dict(
                 table_id='e5c51369-8e71-4ec6-b027-1f92bdc14762',
-                table_version='latest'
+                table_version=0,
```

Comment:
@maxhutch can we expect that version numbers start at 0?

Reply:
Also it seems like "latest" is supported, at least in the api.

Reply:
Good call-out for the version numbers. I'll check and add to the docstring too.

The story ACs include: "The predictor module references a specific version of the file in the data service." and its also in the ai-engine-documentation and was discussed a little here: https://github.com/CitrineInformatics/ai-engine-documentation/pull/43#issuecomment-602800680

I think it would be easy enough to accept a `str` or `int` so the constructor signature doesn't change in the future if we want to support "latest" (and also we would not throw a client error on "latest").  But, for product reasons, I don't think we want to persist "latest" in the predictor configuration; we'd want to convert it to the explicit latest version number.